src/hw/product_registry.hpp

```diff
@@ -454,4 +464,8 @@ static const std::vector<ProductDescription> productRegistry = {
      .productName = "TATLIN.ARCHIVE.S",
      .fans = FanConnectorsS320,
      .sysFanPN = "ASMFAN781103A",
-     .cpuFanPN = "CPUHSN792004A"}};
+     .cpuFanPN = "CPUHSN792004A"},
+    {.pnameRegex = std::regex("NICOLE-X86.*"),
+     .productName = "NICOLE-X86",
+     .fans = FanConnectorsNicoleX86,
+     .sysFanPN = "NODE_CPU_FAN"}};
```

Comment:
Why a system fan PN is set to 'CPU Fan' ? Isn't that what `.cpuFanPN` is for?

Reply:
Based on the results of the discussion with Pavel, it became clear that it is supposed to cool the CPU airflow from system fans, whose PN is not known. But here is a typo, it should be "NODE_SYS_FAN"